packages/webapp/src/components/Finances/FinancesCarrousel/index.jsx

```diff
@@ -0,0 +1,226 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Chart as ChartJS, ArcElement, Tooltip } from 'chart.js';
+import { Doughnut } from 'react-chartjs-2';
+import { useTranslation } from 'react-i18next';
+import { BsChevronRight } from 'react-icons/bs';
+import PropTypes from 'prop-types';
+import CardsCarrousel from '../../CardsCarrousel';
+import { ReactComponent as ProfitLossIconDark } from '../../../assets/images/finance/Profit-loss-icn-dark.svg';
+import { ReactComponent as ProfitLossIconLight } from '../../../assets/images/finance/Profit-loss-icn-light.svg';
+import { ReactComponent as ExpenseIcon } from '../../../assets/images/finance/Expense-icn.svg';
+import { ReactComponent as CropIcon } from '../../../assets/images/finance/Crop-icn.svg';
+import styles from './styles.module.scss';
+import { Semibold, Text } from '../../Typography';
+import clsx from 'clsx';
+import TextButton from '../../Form/Button/TextButton';
+
+ChartJS.register(ArcElement, Tooltip);
+
+const ShadowPlugin = {
+  beforeDraw: (chart) => {
+    const { ctx } = chart;
+    ctx.shadowColor = '#FFEFCF';
+    ctx.shadowBlur = 0;
+    ctx.shadowOffsetX = 0;
+    ctx.shadowOffsetY = 4;
+  },
+};
+
+const FinancesCarrousel = ({
+  totalExpense,
+  totalRevenue,
+  labourExpense,
+  otherExpense,
+  estimatedRevenue,
+  currencySymbol,
+  history,
+}) => {
+  const { t } = useTranslation();
+
+  const expenseChartData = {
+    labels: ['Total other', 'Total labour'],
+    datasets: [
+      {
+        data: [11500, 21523],
```

Comment:
Is the data meant to be fixed so that both colours are always shown on the donut? Or should the chart actually represent `labourExpense` and `otherExpense`?